RELEASENOTES.md

```diff
@@ -1,5 +1,17 @@
 # Release notes
 
+## v2.5.0
+### Improvements
+ - Releasing process involving artifactory (#141)
+ - MuxCore pulled from Maven instead of in bundled jar (#143)
+ - Support for overriding the beacon domain (#138)
+ - Javadoc coverage for public API (#142)
+ - For API version 30+ use context.getDisplay instead of WindowManager. (#145)
+
+### Fixes
+ - Removed VideoComponent listener and now capturing firstFrameRendered … (#144)
```

Comment:
Can we expand on this one so the release notes have a full description and don't require linking to a previous PR please?

Reply:
The `...` was unnecessary, I think this description is sufficient.